src/rpc/ws_client/mod.rs

```diff
@@ -18,6 +18,8 @@
 use crate::rpc::Error as RpcClientError;
 pub use ac_node_api::{events::EventDetails, StaticEvent};
 pub use client::WsRpcClient;
+pub use subscription::WsSubscriptionWrapper;
```

Comment:
I think this is a good example, where the outside does not care if the `WsSubscriptionWrapper` lives inside the subscription module or not.

Reply:
Yes, it is